script/glide.sh

```diff
@@ -49,7 +36,7 @@ if ! type glide-vc > /dev/null 2>&1; then
   exit 1
 fi
 
-if [[ ! -e "${CWD}/glide.yaml" ]]; then
+if [[ ! -e "glide.yaml" ]]; then
```

Comment:
This won't allow us to call the script from anywhere but the root directory.

I suggest to define

```bash
SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
readonly SCRIPT_DIR
readonly GLIDE_DIR="${SCRIPT_DIR}/.."
```

near the top and then use

```bash
if [[ ! -e "${GLIDE_DIR}/glide.yaml" ]]; then
```

on this line.